.github/workflows/ci.yml

```diff
@@ -200,14 +200,14 @@ jobs:
           # install java
           sudo apt update -y && sudo apt install openjdk-11-jdk -y
 
-      - name: Run Minimize Test
+      - name: Run Mini Test
         env:
           GS_TEST_DIR: ${{ github.workspace }}/gstest
         run: |
           # download dataset
           git clone -b master --single-branch --depth=1 https://github.com/7br/gstest.git ${GS_TEST_DIR}
 
-          python3 -m pytest -s -v python/tests/minitest
+          python3 -m pytest -s -v $(dirname $(python3 -c "import graphscope; print(graphscope.__file__)"))/tests
```

Comment:
This grammar is a little bit weird... Why this effort?

Reply:
Need to test graphscope installed by `pip install`, which exists in `/syspath/site-packages/graphscope`